### DISCUSSION TITLE
Build Error since Commit 79d354a

### DISCUSSION THREAD

**Discussion by geolta** [texstudio-org/texstudio#3031] [Q&A] [answered]
Hi, I'm getting an error when trying to build TXS in Qt Creator. I think, this occurs since the commit 79d354a ([update internally build quazip to v1.4](https://github.com/texstudio-org/texstudio/commit/79d354aca4cc3b0faef05ea83e1d448f4a9b38e7)). Do I have to install something additionally?

```
texstudio/src/spellerutility.cpp:14: error: JlCompress.h: No such file or directory
../texstudio/src/spellerutility.cpp:14:10: fatal error: JlCompress.h: No such file or directory
   14 | #include "JlCompress.h"
      |          ^~~~~~~~~~~~~~
```

I'm on Fedora KDE 36 with the latest code from master branch.

**Comment by sunderme** [comment]
simplest solution is to install the quazip package on your system.
Internal quazip is a fall-back.

**Reply by geolta** [reply]
Hm, doesn't seem to work for me. I have installed quazip, quazip-devel, quazip-qt5, quazip-qt5-devel, quazip-qt6, quazip-qt6-devel but the error persists.

Additionally, the build should be able to handle the fallback anyway, shouldn't it?

**Reply by sunderme** [reply]
not sure what happens on your side, but building against quazip works on all platforms, including fedora.
Have you tried a clean build ?

**Reply by geolta** [reply]
Yes, I tried a clean build in various configurations, unfortunately none of them worked. But independently of my special case, the following lines in texstudio.pro are wrong, aren't they? Because `src/quazip/quazip/quazip.pri` indeed doesn't exist.
```
isEmpty(USE_SYSTEM_QUAZIP) {
    include(src/quazip/quazip/quazip.pri)
}
```

**Reply by sunderme** [reply, marked as answer]
qmake is not supported anymore. Use cmake.

**Reply by geolta** [reply]
Aaaaah, yes, reading the documentation helps. This was indeed my mistake, thank you a lot :)
